src/parameters/parameters_ode_solver.cpp

```diff
@@ -289,6 +295,11 @@ void ODESolverParam::parse_parameters (dealii::ParameterHandler &prm)
             else if (output_string_rrk == "quiet")   rrk_root_solver_output = quiet;
 
             relaxation_runge_kutta_root_tolerance = prm.get_double("relaxation_runge_kutta_root_tolerance");
+            use_relaxation_runge_kutta = prm.get_bool("use_relaxation_runge_kutta");
+            if (use_relaxation_runge_kutta == false && ode_solver_type == rrk_explicit_solver) {
+                // For backwards compatibility
+                use_relaxation_runge_kutta = true;
```

Comment:
I could likely remove the `rrk_explicit_solver` before merging this in.

Reply:
Having thought on it more, I have decided it's best to use `use_relaxation_runge_kutta` primarily, but keep backwards compatibility with the `rrk_explicit_solver` version.

Reply:
Sure, not a bad idea! It should be completely safe to select the depreciated rrk_explicit_solver, but just in case I have added a warning.